centrallix/Makefile.in

```diff
@@ -526,8 +526,8 @@ tests/centrallix.conf-test: tests/centrallix.conf-test.in
 	RUNLIBDIR=$${LIBDIR#$(builddir)}; \
 	$(SED) -e "s|##SYSCONFDIR##|$$RUNSYSCONFDIR|g" -e "s|##LIBDIR##|$$RUNLIBDIR|g" -e "s|##LOGMETHOD##|@LOGMETHOD@|g" tests/centrallix.conf-test.in > tests/centrallix.conf-test
 
-tests/test_%.bin: tests/test_%.o .testdepend
-	$(CC) $< tests/t_driver.c $$(sed -n 's#^$@: ##p' < .testdepend) $(LIBS) -o $@
+tests/test_%.bin: tests/test_%.o tests/t_driver.c centrallix.o $(V3LSOBJS) $(STATIC_LIBS)
+	$(CC) $< tests/t_driver.c centrallix.o $(V3LSOBJS) $(LIBDIRS) $(PROFILE) $(COVERAGE) -Wl@EXPORT_DYNAMIC@ -o $@ $(LIBS)
```

Comment:
This is a little pedantic of me, esp. since I "started it", but ideally t_driver.c should be compiled to a .o in a separate makefile rule, which avoids the compiler having to recompile that particular .c file for every single test_%.c.